client/src/i18n/en/posting_journal.json

```diff
@@ -5,6 +5,7 @@
     "ROWS" : "Row(s)",
     "EDITING" : "Editing",
     "NO_TRANSACTION_TYPE" : "No assigned transaction type",
+    "HAS_PREVIOUS_EDITS" : "Last edit by {{ user }} on {{ date }}",
```

Comment:
Why not attach it from the controller? so that we will avoid digest issue
This is a no blocking comment

Reply:
I don't think this should do more than one digest.  Since it uses the `translate` directive, it will only change if the `user` and `date` values change, which they never do.